src/Main.elm

```diff
@@ -104,18 +104,36 @@ view model =
                 , ul [ class "fields__font-list" ]
                     <| List.indexedMap viewFont model
                 ]
-            , pre
-                [ class "main__font-faces" ]
-                [ List.filterMap viewFontFaces model
-                    |> String.join "\n"
-                    |> text
-                ]
+            , viewFontFaces model
+            ]
+        ]
+
+viewFontFaces : Model -> Html Msg
+viewFontFaces model =
+    let
+        selected =
+            List.filterMap pickSelected model
+    in
+        section [ class "font-faces" ]
+            [ viewFontsForPlatform "Web" (web selected)
+            , viewFontsForPlatform "Android" (android selected)
+            ]
+
+viewFontsForPlatform : String -> List String -> Html Msg
+viewFontsForPlatform heading fonts =
```

Comment:
I'd consider calling the second argument something more along the lines of 'font faces', given its type. Super minor point!